### PR TITLE
VIM-4648: Only support Phase 1 notifications

### PR DESCRIPTION
**Pull request by fyell**
## Ticket
https://vimean.atlassian.net/browse/VIM-4648

## Ticket Summary
Notification Model

## Implementation Summary
- Updated enums to support phase 1 notifications as specified here https://docs.google.com/spreadsheets/d/1FlhhF-DqBrsO-iWLRdu0zSCMj-CsZWMsAjbKo9r5cW8/edit#gid=275549178
- Fix key for video parsing in VIMCredit

## How to Test
N/A

### CODE REVIEW

**Comment by fyell** [comment]
build this please

**Comment by cameo-github** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><strong><a href="https://github.com/vimeo/VimeoNetworking/blob/b822868d3fdb5b3fc482a12f6a5d94c5bd22e1c1/VimeoNetworking/Sources/VimeoClient.swift#L322">VimeoNetworking/Sources/VimeoClient.swift#L322</a></strong>: cast from ‘Response&lt;VIMNullResponse&gt;’ to unrelated type ‘Response&lt;ModelType&gt;’ always fails  <br /><code>
                let response = Response(model: nullResponseObject, json: [:]) as! Response&lt;ModelType&gt;
</code></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 22 tests, with 0 failures (0 unexpected) in 0.321 (0.334) seconds</td>
    </tr>
  </tbody>
</table>

See build details on [CircleCI](https://circleci.com/gh/vimeo/VimeoNetworking/126)

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: VimeoNetworking/Sources/Models/VIMNotification.m [hunk 1]
```diff
@@ -55,6 +72,10 @@ - (Class)getClassForObjectKey:(NSString *)key
         return [VIMCredit class];
     }
     
+    if ([key isEqualToString:@"comment"]) {
```
Comment (jenniferlim) [inline VimeoNetworking/Sources/Models/VIMNotification.m]: nit: can we add a newline for the `{`

**Comment by jenniferlim** [comment]
DWR @fyell - Looks good.

**Pull request merged**